ddtrace/contrib/pytest/plugin.py

```diff
@@ -128,26 +139,43 @@ def _get_pytest_command(config):
 
 def _get_module_path(item):
     """Extract module path from a `pytest.Item` instance."""
-    if not isinstance(item, pytest.Package):
+    if not isinstance(item, (pytest.Package, pytest.Module)):
         return None
     return item.nodeid.rpartition("/")[0]
 
 
-def _get_module_name(item):
+def _get_module_name(item, is_package=True):
     """Extract module name (fully qualified) from a `pytest.Item` instance."""
+    if not is_package:
+        return item.nodeid.rpartition("/")[0].replace("/", ".")
```

Comment:
very personal nit: I have this vendetta against `not` in `if` conditions, so I'd flip this around to 
```suggestion
    if is_package:
        return item.module.__name__
    return item.nodeid.rpartition("/")[0].replace("/", ".")       
```